src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,22 @@
 import './MovieList.scss';
+import React from 'react';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ moviesFromServer }) => (
+  <div className="page">
+    <div className="page-content">
+      <div className="movies">
+        {moviesFromServer.map(movie => (
+          <MovieCard
+            moviesFromServer={movie}
```

Comment:
The prop name should be `movie` instead of `moviesFromServer` when passing a movie to the `MovieCard` component, as specified in the task description.